app/workers/comments/bust_cache_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Comments
+  class BustCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
+
+    def perform(comment_id)
+      comment = Comment.find_by(id: comment_id)
+      return unless comment&.commentable
+
```

Comment:
One last thing we need to add to this(you can blame @rhymes 😉) [we added a purge call here in a recently merged PR](https://github.com/thepracticaldev/dev.to/blob/master/app/jobs/comments/bust_cache_job.rb#L9) so that needs to be added here.
```
comment.purge
```
In addition, [in your spec](https://github.com/thepracticaldev/dev.to/pull/5502/files#diff-a33696b7ea87f4f16f9ce40d0a9287ddR20) you will want to stub that purge method by adding this line
```
allow(comment).to receive(:purge)
```

If for some reason you don't have time to add those just let me know and I would be happy to do it!!!

Reply:
No worries at all, I will address this later today